Dockerfile

```diff
@@ -34,6 +35,9 @@ RUN curl -fsSL "$GOLANG_DOWNLOAD_URL" -o golang.tar.gz \
 
 ENV PATH /usr/local/go/bin:$PATH
 
+# install golangci_lint 
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.56.2
```

Comment:
This step need verification since $(go env GOPATH) may be not in system $PATH

Reply:
updated, move to /usr/local/go/bin/